README.md

```diff
@@ -63,6 +68,12 @@ Call `movieArt(function(e){console.log(e);});` or run the CLI command with no ar
 
```

Comment:
Year is not required for either of them, but permitted for both.

Size is afaik also available for both. The API is really practically identical except for that one change from movie to tv